CONTRIBUTING.md

```diff
@@ -85,6 +86,13 @@ A lot of the instructions below replicate what the [PR CI pipeline] is doing.
 1. You are now ready to submit your PR.
 1. After your PR ise merged, most likely you will want to read [`How to deploy your changes`](#how-to-deploy-your-changes) to verify they got deployed.
 
+# New linter rule promotion strategy
+
+1. Merge new rules to main. As a result, they show up in the Staging LintDiff pipeline.
```

Comment:
Thanks for the write-up! Per our chat, let's discuss how to handle baselining, i.e. the problem of existing API versions failing on newly added rules.